spec/models/provider_spec.rb

```diff
@@ -248,4 +260,29 @@
       expect(Provider.opted_in).to match_array([opted_in_provider])
     end
   end
+
+  describe '#updated_changed_at' do
```

Comment:
this name is a bit ambiguous, could we be more explicit? how about "changed_at is updated"